rundeckapp/grails-spa/packages/ui-trellis/src/library/components/widgets/rundeck-info/RundeckInfo.spec.ts

```diff
@@ -108,4 +109,15 @@ describe("RundeckInfo", () => {
     expect(latestReleaseComponent.props().number).toBe("v5.2.0-20240410");
     expect(latestReleaseComponent.props().tag).toBe("GA");
   });
+  it("does not render release information when hideVersion is true", async () => {
+    const wrapper = await mountRundeckInfo({ hideVersion: true });
+    const rundeckVersionComponent = wrapper.findComponent({
+      name: "RundeckVersion",
+    });
+    const versionDisplayComponent = wrapper.findComponent({
+      name: "VersionDisplay",
+    });
```

Comment:
The idea of testing is to check that what is shown to the user is tested, so we need to change this to the right behavior, that some divs inside those components are displayed, so you should change that find elements and then check its existance
```suggestion
    const rundeckVersionComponent = wrapper.find('div.rundeck-version-display');
    const versionDisplayComponent = wrapper.find('span.rundeck-version-icon');
```

Please check if that works if you want to change it, I'm a bit rusty with those kind fo tests 😅